src/Views/batchprocessor.html

```diff
@@ -26,9 +26,20 @@ <h4 class="modal-title">StreamStats Batch Processor</h4>
     <div style="padding-top: 10px;">
         <!-- Start submit batch -->
         <div id="submitBatch" ng-if="vm.selectedBatchProcessorTabName==='submitBatch'">
-           
+            <!-- Start instructions -->
+            <div id="submitBatchInstructions"> To submit a batch: <br>
+                <ul style="list-style-type: circle;">
+                    <li>All highlighted fields must be completed before submission</li>
+                    <li>Only points that are contained within the selected State/Region will be processed</li>
+                    <li>Batches are limited to 250 points each and if it contains more, only the first 250 points will be
+                    processed</li>
+                    <li>Only point shapefiles will be processed so no polygon, polyline, or multipoint geometries</li>
+                </ul>
+            </div>
+            <!-- End instructions -->
+
             <!-- Start select region -->
-            <div id="selectRegionID" style="padding-top: 15px;">
+            <div id="selectRegionID" style="padding-top: 10px;">
                 <label> State / Region: </label>

```

Comment:
I noticed that the <label> tag was making this label appear different from other form elements
```suggestion
                State / Region:

```